aptos-move/move-examples/defi/sources/locked_coins.move

```diff
@@ -107,6 +124,21 @@ module defi::locked_coins {
         });
     }
 
+    /// Batch version of update_lockup.
+    public entry fun batch_update_lockup<CoinType>(
+        sponsor: &signer, recipients: vector<address>, new_unlock_time_secs: u64) acquires Locks {
```

Comment:
what is the usecase here that batch update to the same unlock time instead of a new time for each recipient?

Reply:
The locked coins can be created together for the same lock time.